R/transfer.R

```diff
@@ -0,0 +1,37 @@
+#' Transferred assets among funding account and other accounts
+#'
+#' Exposes the \code{GET /sapi/v1/asset/transfer} endpoint.
+#'
+#' @inheritParams trade-parameters
+#' @inheritParams limit-5-30
+#' @param ... Additional parameters to pass they query
```

Comment:
Please change to "Further arguments passed to or from other methods.".

Reply:
Changed on next commit